selfdrive/car/ford/carstate.py

```diff
@@ -54,7 +54,8 @@ def update(self, cp, cp_cam):
       ret.steerFaultTemporary |= cp.vl["Lane_Assist_Data3_FD1"]["LatCtlSte_D_Stat"] not in (1, 2, 3)
 
     # cruise state
-    ret.cruiseState.speed = cp.vl["EngBrakeData"]["Veh_V_DsplyCcSet"] * CV.MPH_TO_MS
+    is_metric = cp.vl["INSTRUMENT_PANEL"]["METRIC_UNITS"] == 1 if self.CP.carFingerprint not in CANFD_CAR else False
```

Comment:
Signal does not appear to be present on CAN-FD platform (checked `112e4d6e0cad05e1|2023-11-14--08-21-43`) so will keep this for now.